frontends/web/src/routes/exchange/pocket.tsx

```diff
@@ -175,6 +176,85 @@ export const Pocket = ({ code }: TProps) => {
     });
   };
 
+  const handlePaymentRequest = useCallback(async (message: PaymentRequestV0Message) => {
```

Comment:
I think useCallback here doesn't do anything right? It's not used a dep to any other hook.

Reply:
Good catch! Thanks